invenio_files_rest/models.py

```diff
@@ -433,9 +433,16 @@ def delete(cls, bucket_id):
 
     @ensure_unlocked()
     def remove(self):
-        """Permanently remove a bucket and all objects."""
-        # 1) remove all object versions 2) remove bucket.
-        raise NotImplementedError()
+        """Permanently remove a bucket and all objects (including versions).
+
+        Note the method does not remove the associated file instances which
```

Comment:
Both joined as one `.. note::`? I was mostly wondering why one remove method need framed box and the other doesn't when both messages are quite important.